packages/catalog-process/src/model/domain/errors.ts

```diff
@@ -412,3 +414,15 @@ export function eserviceInterfaceDataNotValid(): ApiError<ErrorCodes> {
     title: "EService template interface data not valid",
   });
 }
+
+export function receiveTemplateMissingTenantKindRiskAnalysis(
+  templateId: EServiceTemplateId,
+  tenantId: TenantId,
+  tenantKind: TenantKind
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Template ${templateId} with mode receive is not compatible with tenant ${tenantId} of kind ${tenantKind}, missing risk analysis`,
+    code: "receiveTemplateMissingTenantKindRiskAnalysis",
+    title: "Receive template missing tenant kind risk analysis",
```

Comment:
This message is misleading, it intend that receive mode is related to template,meanwhile it a property of eservice. I would change into : 
"Eservice Receive mode template missing tenant kind for risk analysis"

Reply:
I've updated it with @AsterITA's [suggestion](https://github.com/pagopa/interop-be-monorepo/pull/1628#discussion_r1993912135)